fdbserver/ptxn/test/Driver.actor.cpp

```diff
@@ -78,6 +81,8 @@ std::shared_ptr<TestDriverContext> initTestDriverContext(const TestDriverOptions
 		context->storageTeamIDs.push_back(getNewStorageTeamID());
 	}
 
+	context->commitVersionGap = 10000;
```

Comment:
10000? would knob give you more test config flexibility?

Reply:
This is a test driver so it's ok. Agree that using a random number is better.